keyboards/gmmk/pro/config.h

```diff
@@ -53,13 +57,16 @@
 #define SPI_MOSI_PIN A6
 #define SPI_MISO_PIN A7
 
-#define DRIVER_1_CS B13
-#define DRIVER_2_CS B14
-#define DRIVER_1_EN C13
-#define DRIVER_2_EN C13
+#define SPI_SS_DRIVER_1_PIN B13
+#define ENABLE_DRIVER_1_PIN C13
+#define SPI_SS_DRIVER_2_PIN B14
+#define ENABLE_DRIVER_2_PIN C13
```

Comment:
Please revert.  would prefer to keep the ISSI like config here.

Reply:
Not sure what you mean. Could you please address me to the ISSI driver to see the style?